go-controller/pkg/ovn/egressip.go

```diff
@@ -1071,17 +1083,12 @@ func (oc *Controller) checkEgressNodesReachability() {
 }
 
 func (oc *Controller) isReachable(node *egressNode) bool {
-	reachable := false
-	// check IPv6 only if IPv4 fails
-	if node.v4IP != nil {
-		if reachable = dialer.dial(node.v4IP); reachable {
-			return reachable
+	for _, ip := range node.mgmtIPs {
```

Comment:
With this change, we are no longer giving pref to ipv4 like the code did in the previous implementation. Does it matter?